packages/service-clients/end-to-end-tests/odsp-client/src/test/audience.spec.ts

```diff
@@ -24,14 +24,27 @@ describe("Fluid audience", () => {
 	const connectTimeoutMs = 10_000;
 	let client: OdspClient;
 	let schema: ContainerSchema;
+
+	const credentials = getCredentials();
+
+	if (credentials.length < 2) {
+		throw new Error("Insufficient login credentials provided");
+	}
+
+	const [cred1, cred2] = credentials.slice(0, 2);
+
+	if (!cred1 || !cred2) {
+		throw new Error("Invalid login credentials format");
+	}
+
 	const client1Creds: IOdspLoginCredentials = {
-		username: process.env.odsp__client__login__username as string,
-		password: process.env.odsp__client__login__password as string,
+		email: cred1.email,
+		password: cred1.password,
 	};
```

Comment:
This seems to be copying cred1 to client1Creds - consider just destructuring directly into client1Creds and skipping cred1.